src/main/java/org/gridsuite/modification/server/entities/equipment/creation/TwoWindingsTransformerCreationEntity.java

```diff
@@ -107,151 +107,58 @@ public class TwoWindingsTransformerCreationEntity extends BranchCreationEntity {
-                twoWindingsTransformerCreationInfos.getBusOrBusbarSectionId1(),
-                twoWindingsTransformerCreationInfos.getVoltageLevelId2(),
-                twoWindingsTransformerCreationInfos.getBusOrBusbarSectionId2(),
-                twoWindingsTransformerCreationInfos.getCurrentLimits1() != null ? twoWindingsTransformerCreationInfos.getCurrentLimits1().getPermanentLimit() : null,
-                twoWindingsTransformerCreationInfos.getCurrentLimits2() != null ? twoWindingsTransformerCreationInfos.getCurrentLimits2().getPermanentLimit() : null,
-                twoWindingsTransformerCreationInfos.getConnectionName1(),
-                twoWindingsTransformerCreationInfos.getConnectionDirection1(),
-                twoWindingsTransformerCreationInfos.getConnectionName2(),
-                twoWindingsTransformerCreationInfos.getConnectionDirection2(),
-                phaseTapChangerInfos != null ? phaseTapChangerInfos.getLowTapPosition() : null,
-                phaseTapChangerInfos != null ? phaseTapChangerInfos.getTapPosition() : null,
-                phaseTapChangerInfos != null ? phaseTapChangerInfos.isRegulating() : null,
-                phaseTapChangerInfos != null ? phaseTapChangerInfos.getTargetDeadband() : null,
-                phaseTapChangerInfos != null ? phaseTapChangerInfos.getRegulatingTerminalId() : null,
-                phaseTapChangerInfos != null ? phaseTapChangerInfos.getRegulatingTerminalVlId() : null,
-                phaseTapChangerInfos != null ? phaseTapChangerInfos.getRegulatingTerminalType() : null,
-                phaseTapChangerInfos != null ? phaseTapChangerInfos.getRegulationMode() : null,
-                phaseTapChangerInfos != null ? phaseTapChangerInfos.getRegulationValue() : null,
-                ratioTapChangerInfos != null ? ratioTapChangerInfos.getLowTapPosition() : null,
-                ratioTapChangerInfos != null ? ratioTapChangerInfos.getTapPosition() : null,
-                ratioTapChangerInfos != null ? ratioTapChangerInfos.isRegulating() : null,
-                ratioTapChangerInfos != null ? ratioTapChangerInfos.getTargetDeadband() : null,
-                ratioTapChangerInfos != null ? ratioTapChangerInfos.getRegulatingTerminalId() : null,
-                ratioTapChangerInfos != null ? ratioTapChangerInfos.getRegulatingTerminalVlId() : null,
-                ratioTapChangerInfos != null ? ratioTapChangerInfos.getRegulatingTerminalType() : null,
-                ratioTapChangerInfos != null ? ratioTapChangerInfos.isLoadTapChangingCapabilities() : null,
-                ratioTapChangerInfos != null ? ratioTapChangerInfos.getTargetV() : null,
-                tapChangerSteps,
-                twoWindingsTransformerCreationInfos.getConnectionPosition1(),
-                twoWindingsTransformerCreationInfos.getConnectionPosition2());
-
-        return twoWindingsTransformerCreationEntity;
+    private void assignRatioTapChanger(RatioTapChangerCreationInfos ratioTapChanger) {
```

Comment:
Rename by `ratioTapChangerInfos` ?

Reply:
We are mapping the data to the entity, we can keep it like this. are you ok ?

Reply:
to be consistent with assignPhaseTapChanger:
ratioTapChanger -> ratioTapChangerCreationInfos